test/ReadsUtils_server_test.py

```diff
@@ -1050,22 +1147,44 @@ def test_upload_fail_interleaved_for_single(self):
              'fwd_id': ret['id'],
              'name': 'bar'
              },
-            'Invalid fasta file /kb/module/work/tmp/fwd/Sample5_interleaved' +
-            '.fastq from Shock node ' + ret['id'])
+            'Invalid fasta file /kb/module/work/tmp/fwd/Sample5_interleaved.fastq')
```

Comment:
Same comment as previous review:
Not having the source shock node here makes errors impossible to debug